src/wazuh_db/wdb_parser.c

```diff
@@ -248,7 +248,7 @@ int wdb_parse(char * input, char * output, int peer) {
 
         // Don't perform this check if it's a manager.
         if (agent_id != 0) {
-            if (wdb_global = wdb_open_global(), !wdb_global) {
+            if (wdb_global = wdb_open_global(), !wdb_global || !wdb_global->enabled) {
```

Comment:
I think we should differenciate this two scenarios to print a different DEBUG message clarifying that the DB was opened but is desabled.

Reply:
Fixed